bin/create-test-app.js

```diff
@@ -44,15 +54,48 @@ program
     false
   )
   .action(async (options) => {
+    // helpers
+    const log = (message) => {
+      console.log(`\r\n🧪 ${message}`)
+    }
+
+    const appExec = async (command, args, options = {}) => {
+      const defaults = { cwd: appPath, stdio: "inherit" }
+      await execa(command, args, { ...defaults, ...options })
+    }
+
+    const pnpmDev = async () => {
+      try {
+        await appExec("pnpm", ["run", "dev"])
+      } catch (error) {}
```

Comment:
Should we only ignore them if it's an error caused by a `CTRL+C`? (assuming that's something we can detect)